src/styles/variables.css

```diff
@@ -2,41 +2,44 @@
  * All of these variables are tokens from Forma-36 and may not be adjusted as these
  * are global variables that may affect different places.
  * Forma-36 tokens: https://f36.contentful.com/tokens/color-system
+ * As our customers may use other design libraries, we try to avoid overlapping global
+ * variables by always using the prefix `--f36-` inside this SDK.
+ * 
  */
 
 :root {
   /* color */
-  --blue100: #e8f5ff;
-  --blue200: #ceecff;
-  --blue300: #98cbff;
-  --blue400: #40a0ff;
-  --blue500: #036fe3;
-  --blue600: #0059c8;
-  --blue700: #0041ab;
-  --blue800: #003298;
-  --blue900: #002a8e;
-  --gray100: #f7f9fa;
-  --gray200: #e7ebee;
-  --gray300: #cfd9e0;
-  --gray400: #aec1cc;
-  --gray500: #67728a;
-  --gray600: #5a657c;
-  --gray700: #414d63;
-  --gray800: #1b273a;
-  --gray900: #111b2b;
-  --red200: #ffe0e0;
-  --red800: #7f0010;
-  --color-white: #ffffff;
-  --glow-primary: 0px 0px 0px 3px #e8f5ff;
+  --f36-blue100: #e8f5ff;
```

Comment:
It might be inevitable at some point these variables to diverge from forma. Also if forma later on decides to adopt the `--f36` prefix as well we'll have the same issue (Probably unlikely but you never know).
Maybe we should use a prefix specific to Exp Builder and avoid mentions of forma all together?

Reply:
Mmmh, I'm in favor of sticking to our built-in design library and forcing us to use these colors, spacings, font sizes, etc. - designers and UX engineers will probably agree on this. So, that's why I would like to keep this prefix specific for the F36 tokens we should reuse. I can change it to `--experience-builder-f36` but still I don't want us to ever change those values to avoid such issues again (see the current text size of the SDKs error screen).

Reply:
I would not try to think here too general/ broad but keep it specific as we can quickly change this prefix later on and so far, we don't anticipate any case where we actually want to use those variables outside of the SDK.

Reply:
So if we need to introduce a new variable that is not used in forma later on, would we use `--f36` again or introduce a new prefix then?

Reply:
If we make a decision to change the prefix, let's also make sure that the prefix is short to save a bit on the size of the css file. I know that at this point it is micro optimisation, but it might help us save in the long run

Reply:
Mmmh, I don't anticipate such a scenario but if so, then we should have a non f36 prefix :)

Reply:
Following the conclusion in Slack, I changed the prefix `--exp-builder` and added a note to the section where they are coming from.